Describe wrapperAttr and e.cancel in onShowing for overlays (#2604)
DevExpress/devextreme-documentation: api-reference/10 UI Components/dxOverlay/1 Configuration/onShowing.md

Model data. Available only if Knockout is used.

##### field(e.cancel): Boolean

> Allows you to cancel overlay showing.

Reviewer: - We don't usually use the word "overlay" as a category.
- -ing

```suggestion
Set this field to **true** if you do not want the {WidgetName} to be displayed.
```

> Set this field to **true** if you do not want the {WidgetName} to be displayed.

Reviewer: The "not" particle is easy to miss when reading.

```suggestion
Set this field to **true** if you want to prevent the {WidgetName} from being displayed.
```
